frontend/kick-covid-19/src/pages/Blog.js

```diff
@@ -22,8 +23,11 @@ class Blog extends React.Component {
   render() {
     return (
       <div className="row">
-        {this.state.posts.map((body, id) => (
-          <Post body={body} key={id} />
+        {this.state.posts.map((body, posts, id) => (
```

Comment:
Here and in the state are where I think the error is.

Reply:
yep, I think you're right.  I think you're just missing curly braces to properly destructure the values from the post object passed to your callback.
```suggestion
        {this.state.posts.map(({body, posts, id}) => (
```
map passes each item in the array and the index as args to your callback function (e.g. `posts.map((post, idx) => {...}`).  The way you currently have it, body is being set to the entire post object, posts is being set to the idx and id is undefined since map does not pass a third arg to the callback.  By adding the `{}` around your values you are plucking the body, posts, and id off the object passed to the map callback (e.g. post) in your case.  does that make sense?

Reply:
also, I would rename `posts` to something else => its kind of confusing that posts would themselves contain posts and I'm not quite sure what would be contained by that property at first glance => a better name could improve readability